modules/Emails/language/en_us.lang.php

```diff
@@ -385,4 +385,28 @@
 
     'LBL_LIST_DATE_MODIFIED' => 'Last Modified',
     'LNK_IMPORT_CAMPAIGNS' => 'Import Campaign',
+    
+    // Email Validation Error messages. Typicaly for Email Validation:
+    'ERR_FIELD_FROM_IS_NOT_SET' => 'From field is not set.',
+    'ERR_FIELD_FROM_IS_EMPTY' => 'From field is empty.',
+    'ERR_FIELD_FROM_IS_INVALID' => 'From field is invalid.',
+    'ERR_FIELD_FROM_ADDR_IS_NOT_SET' => 'From address is not set.',
+    'ERR_FIELD_FROM_ADDR_IS_EMPTY' => 'From address is empty.',
+    'ERR_FIELD_FROM_ADDR_IS_INVALID' => 'From address is invalid.',
+    'ERR_FIELD_FROMNAME_IS_NOT_SET' => 'From-name is not set.',
+    'ERR_FIELD_FROMNAME_IS_EMPTY' => 'From-name is empty.',
+    'ERR_FIELD_FROMNAME_IS_INVALID' => 'From-name is invalid.',
+    'ERR_FIELD_FROM_NAME_IS_NOT_SET' => 'From name is not set.',
+    'ERR_FIELD_FROM_NAME_IS_EMPTY' => 'From name is empty.',
+    'ERR_FIELD_FROM_NAME_IS_INVALID' => 'From name is invalid.',
+    'ERR_FIELD_FROM_ADDR_NAME_IS_NOT_SET' => 'From address and name pair is not set.',
+    'ERR_FIELD_FROM_ADDR_NAME_IS_EMPTY' => 'From address and name pair is empty.',
+    'ERR_FIELD_FROM_ADDR_NAME_IS_INVALID' => 'From address and name pair is invalid.',
+    'ERR_FIELD_FROM_ADDR_NAME_DOESNT_MATCH_REGEX' => 'From address and name pair format is incorrect, use "from@emailaddress.org <Name of Person>" format.',
+    'ERR_FIELD_FROM_ADDR_NAME_INVALID_NAME_PART' => 'Invalid name part of From address and name pair.',
+    'ERR_FIELD_FROM_ADDR_NAME_INVALID_EMAIL_PART' => 'Invalid email address part of From address and name pair.',
+    'ERR_FIELD_FROM_ADDR_NAME_INVALID_EMAIL_PART_TO_FILED_FROM' => 'From address and name pair does not match to from name or email address.',
```

Comment:
Looks like a Typo in key: `TO_FILED_ `should be to `TO_FIELD_` ?

Reply:
Thank you @horus68

Reply:
@gymad not me, it was a translator s I had uploaded it to Crowdin too. Usually translators needs to look in the key to find a meaning to a sentence (or to filter other similar sentences).